Configuration/Services.yaml

```diff
@@ -27,6 +27,13 @@ services:
               method: handleEvent
               event: TYPO3\CMS\Filelist\Event\ProcessFileListActionsEvent
 
+    DMK\MkContentAi\Backend\EventListener\NewsRecordEventListener:
+        public: true
+        tags:
+            - name: event.listener
+              method: modifyRecordListActions
+              event: TYPO3\CMS\Recordlist\Event\ModifyRecordListRecordActionsEvent
```

Comment:
Can be omitted, because event is auto-discovered (see https://github.com/TYPO3/typo3/blob/v11.5.41/typo3/sysext/core/Classes/DependencyInjection/ListenerProviderPass.php#L118).